src/bors/command/mod.rs

```diff
@@ -2,6 +2,12 @@ mod parser;
 use crate::github::CommitSha;
 pub use parser::{CommandParseError, CommandParser};
 
+#[derive(Clone, Debug, PartialEq)]
+pub enum Parent {
```

Comment:
It would be nice to add some documentation here, to explain what `Last` means (i.e. the parent of the last try build on the same PR).